pkg/project/project_test.go

```diff
@@ -16,6 +16,80 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 )
 
+func TestCreate(t *testing.T) {
+	tests := []struct {
+		name        string
+		wantErr     bool
+		projectName string
+	}{
+		{
+			name:        "Case 1: project name is given",
+			wantErr:     false,
+			projectName: "project1",
+		},
+		{
+			name:        "Case 2: no project name given",
+			wantErr:     true,
+			projectName: "",
+		},
+	}
+
```

Comment:
Can you add a case where we have a incorrect project name. For example project name in caps ```TEST1234```

Reply:
```
$ odo project create Test123456
 ✗  ProjectRequest.project.openshift.io "Test123456" is invalid: metadata.name: Invalid value: "Test123456": a DNS-1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name',  or '123-abc', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?')
```

Reply:
https://github.com/openshift/odo/issues/2314#issuecomment-545304018

Reply:
> Can you add a case where we have a incorrect project name. For example project name in caps `TEST1234`

We will cover this scenario in integration test as its not getting validated by fake client.

https://github.com/openshift/odo/issues/2314